Description: Revert ElementaryOS specifics
 ElementaryOS has a couple of specific conventions
 that is not required for the Debian package:
 1. The name of the executable that would be built through
 the upstream source is in the format enforced by
 Elementary for its AppCenter.  This patch changes
 the executable name to a regular recognisable name.
 2. ElementaryOS requires applications distributed through
 its AppCenter to have a application desktop entry.
 For DesktopFolder it is unnecessary since there is no graphical
 means to stop or start / user interact with the application i.e. it
 runs unattended via an autostart file.  This patch removes
 the application desktop file installation.
Author: David Mohammed <fossfreedom@ubuntu.com>
Origin: other
Forwarded: not-needed
Last-Update: 2018-12-04

Index: desktopfolder-1.0.11/data/meson.build
===================================================================
--- desktopfolder-1.0.11.orig/data/meson.build	2018-12-06 10:39:51.365883744 +0000
+++ desktopfolder-1.0.11/data/meson.build	2018-12-06 10:39:51.333883744 +0000
@@ -17,10 +17,6 @@
     install_dir: '/etc/xdg/autostart'
 )
 
-install_data(
-    meson.project_name() + '.desktop',
-    install_dir: join_paths(get_option('datadir'), 'applications')
-)
 
 install_data(
     meson.project_name() + '.appdata.xml',
Index: desktopfolder-1.0.11/data/com.github.spheras.desktopfolder.desktop
===================================================================
--- desktopfolder-1.0.11.orig/data/com.github.spheras.desktopfolder.desktop	2018-12-06 10:39:51.365883744 +0000
+++ desktopfolder-1.0.11/data/com.github.spheras.desktopfolder.desktop	2018-12-06 10:41:00.633883744 +0000
@@ -26,7 +26,7 @@
 Comment[pl]=Zarządzaj pulpitem za pomocą paneli, notatek i obrazów
 Comment[pt_BR]=Organize sua área de trabalho com painéis, notas e fotos
 Comment[tr]=Masaüstünüzü panel, not ve fotoğraflarla düzenleyin
-Exec=com.github.spheras.desktopfolder
+Exec=desktopfolder
 Icon=com.github.spheras.desktopfolder
 Keywords=desktop;folder;file;organize;
 Terminal=false
